lib/model-helper.js

```diff
@@ -226,6 +226,12 @@ var modelHelper = module.exports = {
 
       if (arrayItem) {
         if(typeof arrayItem === 'object') {
+          if (arrayItem.enum) {
+            out.allowableValues = {
+              valueType: 'LIST',
```

Comment:
@shelbys I stole it from `swagger-ui`...... https://raw.githubusercontent.com/swagger-api/swagger-ui/master/dist/swagger-ui.js. We are generating the swagger config part here ourselves, instead of letting `swagger-core` generate it from swagger-spec for us.